### ISSUE TITLE
Choose a Python linter and formatter

### ISSUE THREAD

**Issue by banunkers** [rpa-tomorrow/substorm-nlp#6]
Would be nice to have from the start which we can just include in the conda dependencies once Aron's branch is merged to master.

I've only used [black](https://github.com/psf/black) as a formatter and I suggest we use it as well. [Pylint](https://pypi.org/project/pylint/) is also a nice linter.

**Comment by banunkers** [comment]
Should maybe also consider [isort](https://github.com/PyCQA/isort) for sorting imports.
